sdk/storage/azure-storage-blob/tests/test_get_blob_async.py

```diff
@@ -161,11 +163,36 @@ async def test_get_blob_to_bytes_async(self, resource_group, location, storage_a
         blob = self.bsc.get_blob_client(self.container_name, self.byte_blob)
 
         # Act
-        content = await (await blob.download_blob(max_concurrency=2)).readall()
+        content = await (await blob.download_blob()).readall()
```

Comment:
if set SKIP_LIVE_RECORDING=False then this test will fail (test_get_blob_to_bytes_async is the test name)